demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -261,8 +308,10 @@ def process_image(
             run_isolated,
         )
         is_image_powershell = any(obj.is_powershell for _, obj in files_with_objects)
-
-        dev_image = devtest_image(image, is_image_powershell, self.context.dry_run)
+        mypy_additional_dep = self.base_hook["name"].startswith("mypy-in-docker")
```

Comment:
Please add a comment in code, with the jira task for using a `Hook`-like, easily customizable hooks for `DockerHook`s too